### PR TITLE
feat: added new atom-slider

### PR DESCRIPTION
**Pull request by shravan-n**
Added new atom - Slider along with tests and stories

### CODE REVIEW

File: frontend/src/components/atoms/Slider/index.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { Box, Slider as MuiSlider, styled } from '@mui/material'
+import { theme } from '../../../Themes/theme'
+
+interface SliderProps {
```
Comment (Thota-Aakanksha) [inline frontend/src/components/atoms/Slider/index.tsx:5]: you can extend the muiSliderProps and pass them

Comment (shravan-n) [reply frontend/src/components/atoms/Slider/index.tsx]: Done

File: frontend/src/components/atoms/Slider/index.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  },
+})
+
+const Slider = (props: SliderCustomProps) => {
+  return (
+    <StyledBox data-testid="slider">
+      <StyledSlider
+        defaultValue={props.value}
+        max={props.max}
+        onChange={props.handleChange}
```
Comment (Thota-Aakanksha) [inline frontend/src/components/atoms/Slider/index.tsx:39]: please check the props and send the props appropriately

Comment (shravan-n) [reply frontend/src/components/atoms/Slider/index.tsx]: Done changes

**Review by Thota-Aakanksha — APPROVED** [review]

**Pull request merged**